### PR TITLE
Add iframe docs

### PR DESCRIPTION
**Pull request by huynhr**

### CODE REVIEW

File: pages/iframe/custom.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+project has a custom api that sorts the order of images based on what is equipped by the owner.
+
+## Steps
+
+1. Fork and clone the [iframe](https://github.com/tokenbound/iframe) repository:
+   `git clone git@github.com:githubUser/iframe.git`
+2. Run install with `pnpm`, `yarn`, or `npm`.
+3. Create a `.env.local` file and copy the variables from `.env.example`
+4. Create an account for [Alchemy](https://dashboard.alchemy.com/) and create an API key.
+5. Add in the the Alchemy key to the `NEXT_PUBLIC_ALCHEMY_KEY` and `NEXT_PUBLIC_PROVIDER_ENDPOINT` env variables. `NEXT_PUBLIC_PROVIDER_ENDPOINT` will be the `https` link.
```
Comment (bjfresh) [inline pages/iframe/custom.mdx:17]: Double 'the' here

File: pages/iframe/custom.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+6. Create an account for [n.xyz](https://n.xyz/) and api key. This is for checking token approvals.
+7. Add in the n.xyz api key to `NEXT_PUBLIC_NXYZ_API_KEY`. Make sure you enable cors for `n.xyz` for the api key to your domain or `*` for all domains.
+8. Now add in your custom api endpoint to fetch your NFT's assets to `NEXT_PUBLIC_NFT_ENDPOINT`. The api should return an array of strings as image links. Or just a string as an image link. Without this env variable, the app will assume you want your images fetched from alchemy.
+9. To test this you can put in your the url `/contractAddress/tokenId` when running `pnpm dev` | `yarn dev` | `npm run dev`.
```
Comment (bjfresh) [inline pages/iframe/custom.mdx:21]: `in your the`

Comment (bjfresh) [inline pages/iframe/custom.mdx:21]: @huynhr to be clear, you're saying that the endpoint can be tested at ex. 

```https://localhost:3000/0x..../15```

?

Comment (bjfresh) [reply pages/iframe/custom.mdx:21]: (Or whatever port number the app runs on)

File: pages/iframe/introduction.mdx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Introduction
+
+The iframe allows NFT collections to take advantage of OpenSea's [animation_url](https://docs.opensea.io/docs/metadata-standards) metadata param. Basically you can
+embed a web app in the detailed view of a NFT collection on OpenSea. The iframe is important for projects looking to use tokenbound accounts(TBA) for a number of reasons:
+
+1. View your TBA
+2. View assets such as ERC-721, ERC-1155, ether and ERC-20 tokens
+3. Check if any ERC-721, or ERC-1155 tokens have global or token level approvals. This important information for users who are looking to buy an NFT and the contents within it's TBA, as tokens with pending approvals could have potential be sold right before the NFT is purchased.
```
Comment (bjfresh) [inline pages/iframe/introduction.mdx:8]: it's -> its

Comment (bjfresh) [reply pages/iframe/introduction.mdx:8]: This ```is``` important information for users

potential ```to``` be sold

File: pages/iframe/url.mdx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# URL Implementation
+
+The url method is recommended for projects who have a static image(s) and or have an existing project and wish to show their NFT's TBA.
```
Comment (bjfresh) [inline pages/iframe/url.mdx:3]: projects ```with``` a static image

Comment (bjfresh) [reply pages/iframe/url.mdx:3]: and/or

File: pages/iframe/url.mdx [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+All you have to do is update your metadata server and add the following this url `https://iframe-tokenbound.vercel.app/<contractAddress>/<tokenId>` as your `animation_url`.
+
+For example, here's an example with Sapienz: [https://iframe-tokenbound.vercel.app/0x26727ed4f5ba61d3772d1575bca011ae3aef5d36/1](https://iframe-tokenbound.vercel.app/0x26727ed4f5ba61d3772d1575bca011ae3aef5d36/1)
+
+Any updates to the iframe's design will be automatically happen and you will not have to worry about integrating the new updates.
```
Comment (bjfresh) [inline pages/iframe/url.mdx:8]: will ~be~ automatically happen

File: pages/iframe/url.mdx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# URL Implementation
+
+The url method is recommended for projects who have a static image(s) and or have an existing project and wish to show their NFT's TBA.
+All you have to do is update your metadata server and add the following this url `https://iframe-tokenbound.vercel.app/<contractAddress>/<tokenId>` as your `animation_url`.
```
Comment (bjfresh) [inline pages/iframe/url.mdx:4]: add the following ~this~ url

File: pages/iframe/custom.mdx (hunk 2 above)
Comment (DruzyDowns) [reply pages/iframe/custom.mdx:21]: Yeah I would flip this around to say something more explicit like
````
To test, start your local development server using pnpm dev... etc
then navigate to https://localhost:3000/0x..../[tokenID] to view the token
```

File: pages/iframe/introduction.mdx [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# Introduction
+
+The iframe allows NFT collections to take advantage of OpenSea's [animation_url](https://docs.opensea.io/docs/metadata-standards) metadata param. Basically you can
```
Comment (DruzyDowns) [inline pages/iframe/introduction.mdx:3]: I would remove language in this casual voice and just tryo to be as objective and direct as possible. Take out words like 'basically' and any filler words.

File: pages/iframe/url.mdx [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+
+For example, here's an example with Sapienz: [https://iframe-tokenbound.vercel.app/0x26727ed4f5ba61d3772d1575bca011ae3aef5d36/1](https://iframe-tokenbound.vercel.app/0x26727ed4f5ba61d3772d1575bca011ae3aef5d36/1)
```
Comment (DruzyDowns) [inline pages/iframe/url.mdx:6]: redundant use of 'example'

File: pages/iframe/url.mdx (hunk 6 above)
Comment (DruzyDowns) [inline pages/iframe/url.mdx:4]: Remove casual language? 
'All you have to do is'

**Review by DruzyDowns — COMMENTED** [review]
I think the whole introduction section could use a little cleanup with the way the info is presented and the voice it is written in. 
Screenshot 2023-06-28 at 5 12 22 PM

I would also just collapse all of these different markdown pages into one page with sections. At the very least the page titled Introduction doesn't feel necessary, it could just be at the top of the URL section.

**Comment by bjfresh** [comment]
Closing, as a separate PR has taken care of iFrame docs.

**Pull request closed without merging**